docs/integrations/airflow/extractors/default_extractors.md

```diff
@@ -0,0 +1,362 @@
+---
+sidebar_position: 1
+---
+
+# Default extractors
+
+Default extractors are a new way in Airflow 2.3+ and OpenLineage 0.17.0+ to easily add lineage to your data pipelines by modifying your Airflow operators directly. This means custom operators—built in house or forked from another project—can provide you and your team with lineage data without having to modify the OpenLineage project directly or set any environment variables.
+
+The default extractor works a bit differently under the hood than other extractors. While extractors in the OpenLineage project have a getter method for operator names that they’re associated with, the default extractor looks for two specific methods in the operator itself and will call them directly if found. This means that the implementation for the extractor is just two methods in your operator.
+
+Those methods are `get_openlineage_facets_on_start()` and `get_openlineage_facets_on_complete()`. Either, or both, of the methods may be implemented by the operator.
```

Comment:
This might seem overly verbose, but would it be worth explicitly saying when during the Airflow run cycle these are called? I assume on_start is called when the task is started and on_complete is called when the task is done?